app/steps/many_members_step.rb

```diff
@@ -1,13 +1,16 @@
 class ManyMembersStep < Step
   step_attributes :members
 
-  def valid?
+  validate :members_valid
+
+  def members_valid
```

Comment:
instead of having default behavior, should we just `raise NotImplementedError` here? as far as I can tell, we are not using the default behavior defined in this parent step in any of its children